ecosystem/typescript/sdk/src/token_client.ts

```diff
@@ -5,6 +5,7 @@ import { AptosClient } from "./aptos_client";
 import * as TokenTypes from "./token_types";
 import * as Gen from "./generated/index";
 import { HexString, MaybeHexString } from "./hex_string";
+import { BCS, TxnBuilderTypes } from "./transaction_builder";
 
 const NUMBER_MAX: number = 9007199254740991;
```

Comment:
Could we also update this to MAX_U64_BIG_INT? previously, I don't know TS also support bigint.

Reply:
sure. I can update this